squad/core/tasks/__init__.py

```diff
@@ -222,7 +226,20 @@ def __call__(testrun):
         testrun.status_recorded = True
         testrun.save()
 
-        ProjectStatus.create_or_update(testrun.build)
+        status = ProjectStatus.create_or_update(testrun.build)
+        if status:
+            try:
+                notify_project_status.delay(status.id)
+            except OSError as e:
```

Comment:
I assume it was in the celery worker process? I don't think there is much of a point in handling that there ...